package.json

```
@@ -13,20 +13,20 @@
   },
   "dependencies": {
     "@svgr/webpack": "5.4.0",
-    "autoprefixer": "10.3.6",
+    "autoprefixer": "^10.4.0",
```

```suggestion
    "autoprefixer": "10.4.0",
```
can we pin these dependencies instead and upgrade only when necessary?

---

resolved